automation/test_defaults.sh

```diff
@@ -33,7 +33,7 @@ readarray -t oss < <( \
 for os in "${oss[@]}"; do
   defaults=$(oc get template -l "$os,template.kubevirt.io/default-os-variant = true,$ver_label" -o name | wc -l)
 
-  if [[ $defaults -eq 1 ]]; then
+  if [[ $defaults -eq 1 || "$os" =~ (^os.template.kubevirt.io/rhel6.* '=' true) ]]; then
```

Comment:
This does not look right. Without any rhel6 templates, there shouldn't be any `os.template.kubevirt.io/rhel6.*` labels.

Reply:
Recap of offline talk: We decided to exclude from the defaullt check the templates which are not part of the current build. Thanks!